CMakeLists.txt

```diff
@@ -689,11 +689,18 @@ if(BUILD_TESTS)
     PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/membership.py
     CONSENSUS cft
   )
-
-  # Higher snapshot interval as the test currently assumes that no transactions
-  # are emitted while partitions are up. To be removed when
-  # https://github.com/microsoft/CCF/issues/2577 is implemented
-  set(PARTITIONS_TEST_ARGS --snapshot-tx-interval 10000)
+  set(PARTITIONS_TEST_ARGS
+      # Higher snapshot interval as the test currently assumes that no
+      # transactions
+      # are emitted while partitions are up. To be removed when
+      # https://github.com/microsoft/CCF/issues/2577 is implemented
+      --snapshot-tx-interval
+      10000
+      # Partitions test waits for a lot of elections, so reduce the timeout to
+      # complete faster
+      --election-timeout-ms
+      2000
```

Comment:
This might be an overly-optimistic change, but would be a nice time saving. Let's see what the CI thinks.

Reply:
Could we change the default value in `e2e_args.py` so that it applies to all the tests?

Reply:
I'd rather not. I assume something obscure will fail, for not much speed-up benefit.

Reply:
In fact even this change is causing the Debug build to occasionally timeout, so I've removed it.